app/src/test/java/com/kickstarter/viewmodels/DashboardRewardStatsRowHolderViewModelTest.java

```diff
@@ -0,0 +1,91 @@
+package com.kickstarter.viewmodels;
+
+import android.support.annotation.NonNull;
+import android.util.Pair;
+
+import com.kickstarter.KSRobolectricTestCase;
+import com.kickstarter.factories.ProjectFactory;
+import com.kickstarter.factories.ProjectStatsEnvelopeFactory;
+import com.kickstarter.libs.Environment;
+import com.kickstarter.libs.utils.NumberUtils;
+import com.kickstarter.models.Project;
+import com.kickstarter.services.apiresponses.ProjectStatsEnvelope;
+
+import org.junit.Test;
+
+import rx.observers.TestSubscriber;
+
+public class DashboardRewardStatsRowHolderViewModelTest extends KSRobolectricTestCase {
+  private DashboardRewardStatsRowHolderViewModel.ViewModel vm;
+
+  private final TestSubscriber<String> rewardBackerCount = new TestSubscriber<>();
+  private final TestSubscriber<String> rewardMinimum = new TestSubscriber<>();
+  private final TestSubscriber<String> percentageOfTotalPledged = new TestSubscriber<>();
+  private final TestSubscriber<Pair<Project, Float>> projectAndPledgedForReward = new TestSubscriber<>();
+
+  protected void setUpEnvironment(final @NonNull Environment environment) {
+    this.vm = new DashboardRewardStatsRowHolderViewModel.ViewModel(environment);
+    this.vm.outputs.rewardBackerCount().subscribe(this.rewardBackerCount);
+    this.vm.outputs.rewardMinimum().subscribe(this.rewardMinimum);
+    this.vm.outputs.percentageOfTotalPledged().subscribe(this.percentageOfTotalPledged);
+    this.vm.outputs.projectAndPledgedForReward().subscribe(this.projectAndPledgedForReward);
+  }
+
+  @Test
+  public void testRewardBackerCount() {
+    final Project project = ProjectFactory.project();
+    final ProjectStatsEnvelope.RewardStats rewardStats = ProjectStatsEnvelopeFactory.RewardStatsFactory
+      .rewardStats()
+      .toBuilder()
+      .backersCount(10)
+      .build();
+
+    setUpEnvironment(environment());
+    this.vm.inputs.projectAndRewardStats(Pair.create(project, rewardStats));
```

Comment:
can just use `ProjectFactory.project()` here since we're not asserting on or using `project` elsewhere in the test